module/forms/validator/v1.js

```diff
@@ -0,0 +1,15 @@
+module.exports = (req) => {
+	let formsValidator = {
+		create: function () {
+			req.checkBody('type').exists().withMessage('required form type')
+			req.checkBody('subType').exists().withMessage('required form subType')
```

Comment:
@Prajwal17Tunerlabs retain this check
.matches(/^[A-Za-z]+$/)
			.withMessage('type is invalid')

Reply:
made changes to validator